deepparse/network/seq2seq.py

```diff
@@ -9,10 +9,10 @@
 import torch
 from torch import nn
 
-from .decoder import Decoder
-from .encoder import Encoder
-from .. import handle_weights_upload
-from ..tools import download_weights, latest_version
+from deepparse.network.decoder import Decoder
+from deepparse.network.encoder import Encoder
+from deepparse.weights_tools import handle_weights_upload
+from deepparse.download_tools import download_weights, latest_version
```

Comment:
If we want to change to an absolute path, all the library code should use absolute. For this PR, I think it would be more clear to keep using a relative path and, in a subsequent PR, change it to absolute.